mitmproxy/tools/web/app.py

```diff
@@ -221,6 +220,11 @@ def __init_subclass__(cls, **kwargs):
             if fn is not tornado.web.RequestHandler._unimplemented_method:
                 setattr(cls, method, AuthRequestHandler._require_auth(fn))
 
+    def get_auth_cookie_name(self):
+        _, port = tornado.httputil.split_host_and_port(self.request.host)
+        port = port if port is not None else 80
```

Comment:
The negated conditional is a bit hard to read.

Either `port = 80 if port is None else port`, or remove it entirely and `return f"mitmproxy-auth-{port or 80}"`.